server/src/internalClusterTest/java/org/elasticsearch/search/SearchCancellationIT.java

```diff
@@ -291,7 +293,7 @@ public void testCancellationDuringTimeSeriesAggregation() throws Exception {
                 bulkRequestBuilder.add(
                     client().prepareIndex("test")
                         .setOpType(DocWriteRequest.OpType.CREATE)
-                        .setSource("@timestamp", now + (long) i * numberOfDocsPerRefresh + j, "val", (double) j, "dim", String.valueOf(i))
+                        .setSource("@timestamp", now + (long) i * numberOfDocsPerRefresh + j, "val", (double) j, "dim", String.valueOf(j))
```

Comment:
So we end up with thousands of dimensions? Do we want that shape here?

Reply:
I don't think it hurts, but I will reduce it to 100 dimensions.

Reply:
You could randomize it if you want. I'm ok with whatever. So long as I know the actual fix is the other part.

Reply:
The number of shards and document is already randomized, so it should be ok. The goal here is to ensure we have enough different ids to ensure more or less equal distribution of documents across the shards.